card_class.py

```diff
@@ -0,0 +1,110 @@
+"""
+Module card2.
+Definit une classe Card et un petit cas-test qui demontre son utilite.
+TODO: 
+  - coder la fonction str
+  - le init qui prend 0 ou 2 parametres (indication : chercher sur StackOverflow "python optional arguments in initializer of python class")
+  - coder points ; d'ailleurs, est-ce une fonction ou un attribut ? 
+  - corriger le bug : qui est que le 10 est plus fort que Dame Roi Valet (7 8 9 V D R 10 As) 
+  - bonus : ajouter les atouts
+  - bonus : ajouter un setter qui empeche couleur = "VERTE"
+"""
+from enum import Enum
+
+COEUR, CARREAU, PIQUE, TREFLE = (0, 1, 2, 3)
+SEPT, HUIT, NEUF, DIX, VALET, DAME, ROI, AS = (7, 8, 9, 10, 11, 12, 13, 14)
+
+class Color(Enum):
+	COEUR = 0
+	CARREAU = 1
+	PIQUE = 2
+	TREFLE = 3
+	
+class Valeur(Enum):
+	SEPT = 7
+	HUIT = 8
+	NEUF = 9
+	DIX = 10
+	VALET = 11
+	DAME = 12
+	ROI = 13
+	AS = 14
+
+class Card:
+	def __init__(self, valeur = None, couleur = None):
+		self.couleur = couleur
+		self.valeur = valeur
+		self.points = 0
+
+
+	def __str__(self):
+		p = str(Color(self.couleur).name)  #FIXME
+		q = str(Valeur(self.valeur).name)
+		return (q + " de " + p)
+        
+	def points_carte (self):
+		if (self.valeur == 7) or (self.valeur == 8) or (self.valeur == 9):
+			self.points = 0
+		if self.valeur == DIX :
+			self.points = 10
+		if self.valeur == VALET:
+			self.points = 2
+		if self.valeur == DAME :
+			self.points = 3
+		if self.valeur == ROI :
+			self.points = 4
+		if self.valeur ==  AS :
+			self.points = 11
+		
+        
+    
+
+card1 = Card(ROI, PIQUE)
+card1.points_carte()
+# ~ card1 = Card()
+# ~ card1.couleur = Color.COEUR
+# ~ card1.valeur = 10
+
+# card1.couleur = "VERTE"
+
+
+card2 = Card(10, PIQUE)
+card2.points_carte()
+# ~ card2.valeur = 9
+# card2.couleur = Color.COEUR
+
+#
```

Comment:
Commentaires et appels à points_carte en trop
```suggestion
card1 = Card(ROI, PIQUE)
card2 = Card(10, PIQUE)
```